wm/frame.go

```diff
@@ -184,29 +180,29 @@ func (f *frame) getGeometry() (int16, int16, uint16, uint16) {
 	return f.x, f.y, f.width, f.height
 }
 
-func (f *frame) updateGeometry(x, y int16, w, h uint16) {
-	resize := w != f.width || h != f.height
-	move := x != f.x || y != f.y
-	if !move && !resize {
-		return
+func (f *frame) updateGeometry(x, y int16, w, h uint16, force bool) {
```

Comment:
Passing bool here is indicating that the method does two things.
Perhaps a new method, "updateGeometryIfChanged" would do such a check, then call updateGeometry if there is a change?
This keeps the API a little clearer

Reply:
True - I followed the applyTheme method here of using force bool.  There are cases where we want to force a  geometry update (because otherwise it would determine the window has not moved or been resized and thus not apply geometry updates to the internal window).  What would you like to do with this?  Break it out into two methods as you suggest?  Or follow the pattern of applyTheme()?